<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar</string>
  </property>
  <widget class="QLineEdit" name="Cedula">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cedula</string>
   </property>
  </widget>
  <widget class="QPushButton" name="consultar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_consultar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <row/>
   <column>
    <property name="text">
     <string>Tipo grafica</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="graficar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Consultar historial</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>471</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="plano"/>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de grafica:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoGrafica">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Correlacion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Periodograma</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Histograma</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Segmento</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fechaConsulta">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
